team.go

```diff
@@ -165,3 +185,21 @@ func (api *Client) GetBillableInfoForTeamContext(ctx context.Context) (map[strin
 
 	return api.billableInfoRequest(ctx, "team.billableInfo", values)
 }
+
+func (api *Client) GetAuthTeams() ([]TeamInfoShort, error) {
+	return api.GetAuthTeamsContext(context.Background())
+}
+
+func (api *Client) GetAuthTeamsContext(ctx context.Context) ([]TeamInfoShort, error) {
```

Comment:
Could you support `include_icon` param?

https://api.slack.com/methods/auth.teams.list